torch_geometric/utils/homophily.py

```diff
@@ -37,17 +37,30 @@ def homophily(edge_index: Adj, y: Tensor, batch: OptTensor = None,
 
       That measure is called the *node homophily ratio*.
 
+    - In the "Large-scale learning on non-homophilous graphs: \
```

Comment:
Can we make this a link that points to the paper (similar to the other bullet points)?